packages/dialog/src/vaadin-dialog-base-mixin.d.ts

```diff
@@ -38,4 +38,28 @@ export declare class DialogBaseMixinClass {
    * @attr {string} overlay-role
    */
   overlayRole: string;
+
+  /**
+   * Set the distance of the overlay from the top of its container.
+   * If a unitless number is provided, pixels are assumed.
+   *
+   * Note that the overlay top edge may not be the same as the viewport
+   * top edge (e.g. the Lumo theme defines some spacing to prevent the
+   * overlay from stretching all the way to the top of the viewport).
+   *
+   * @type {string}
```

Comment:
nit: we don't really need `@type` annotations in `.d.ts`. Actually IIRC we don't need them in `.js` files too, they were added back in the day when we used to have a script to generate `.d.ts` as part of `magi` release.

Reply:
Done.